stdlib/flux_test.go

```diff
@@ -0,0 +1,24 @@
+// DO NOT EDIT: This file is autogenerated via the builtin command.
```

Comment:
Why do we need to generate this file? Does it change if any of the test cases change? While possible to generate it I wonder if it adds unnecessary complexity.  As such I think we should just commit this file as is, instead of generating it. Then we can add to it without having to update the generating code.

Reply:
This file does change if the test cases change. In particular if a new testing package is added, the new import will be added to this file and a new call to `runEndToEnd` added. This way any modifications to the flux tests won't require changes to how they are run. That will automatically be taken care of. But I don't like complexity either, so I'm totally fine having it be a manual process for now too.

Reply:
What if we generated a file that created a list of test packages. Then we can generate a simple file that is just a bunch of append calls. Then the test file itself is hand written to use that list.

Reply:
That's a lot better because then influxDB can use the same file for it's test runner.